cypress/integration/capture.spec.js

```diff
@@ -91,6 +105,7 @@ describe('Event capture', () => {
             cy.wait(50)
             cy.get('[data-cy-custom-event-button]').click()
             cy.phCaptures().should('deep.equal', ['$pageview', 'custom-event'])
+            cy.get('@consoleError').should('not.be.called')
```

Comment:
I wonder if we should be doing this more extensively in our Cypress tests to figure out if something unknown is breaking down